default.yml

```diff
@@ -46,17 +41,6 @@ Layout/MultilineOperationIndentation:
 Lint/SuppressedException:
   Enabled: false
 
-Layout/SpaceInsideBlockBraces:
-  # The space here provides no real gain in readability while consuming
-  # horizontal space that could be used for a better parameter name.
-  # Also {| differentiates better from a hash than { | does.
-  SpaceBeforeBlockParameters: false
-
-# No trailing space differentiates better from the block:
-# foo} means hash, foo } means block.
-Layout/SpaceInsideHashLiteralBraces:
-  EnforcedStyle: no_space
```

Comment:
I have never had trouble differentiating between blocks and hashes, and no space feels cramped

```ruby
# no space
hash = {first_name: "Tom", last_name: "Brooks"}

# default
hash = { first_name: "Tom", last_name: "Brooks" }
```

Reply:
Same as the previous comments, I would much rather have space.

Reply:
Same as [1]

Reply:
I prefer the space but I'm good either way since this is an auto-fix.